### PR TITLE
Resurface worker status

### PR DESCRIPTION
**Pull request by jaclynperrone**
This PR does a couple things:

- Resurfaces the "failed" and "uploading..." worker statuses. They were overlooked and hidden during the redesign :( But now they are back! :) 
- Separates the worker statuses from the linked BAMs into two different columns. Originally they were sitting in the same table cell, but they are now separated because they are 2 different types of information.

screen shot 2015-09-14 at 4 30 37 pm

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/cycledash/838)

### CODE REVIEW

File: cycledash/static/js/runs/components/RunsPage.js [hunk 1]
```diff
@@ -362,17 +364,15 @@ var RunLabels = React.createClass({
...
       {path: ['fail'], title: 'Has a failed worker task', cssClass: 'fail'}
     ];
     var labels = labelSpecs.map(
       function({path, usePathValue, title, cssClass}) {
-        var value = utils.getIn(run, path) || utils.getIn(taskStates, path);
+        var value = utils.getIn(taskStates, path);
         if (value) {
           return (
-            <span className={`linked-bam ${cssClass}`} title={title} key={path.join('-')}>
+            <span className={`${cssClass}`} title={title} key={path.join('-')}>
```
Comment (ihodes) [inline cycledash/static/js/runs/components/RunsPage.js]: `${cssClass}` => cssClass

File: cycledash/static/js/runs/components/RunsPage.js [hunk 2]
```diff
@@ -250,7 +250,8 @@ var RunsTable = React.createClass({
             <th className='caller-name'>Caller Name</th>
             <th className='date'>Submitted</th>
             <th className='num-variants'>Variants</th>
-            <th className='bam-info'>Linked BAMs</th>
+            <th className='linked-bams'>Linked BAMs</th>
+            <th className='run-labels'></th>
```
Comment (ihodes) [inline cycledash/static/js/runs/components/RunsPage.js]: I'd name this and the corresponding React component something like TaskLabels to reflect its new, more limited scope.

File: cycledash/static/js/runs/components/RunsPage.js [hunk 3]
```diff
@@ -386,6 +386,36 @@ var RunLabels = React.createClass({
   }
 });
 
+var RunBAMs = React.createClass({
```
Comment (ihodes) [inline cycledash/static/js/runs/components/RunsPage.js]: Maybe LinkedBams?

File: cycledash/static/js/runs/components/RunsPage.js [hunk 4]
```diff
@@ -386,6 +386,36 @@ var RunLabels = React.createClass({
...
+  propTypes: {
+    run: React.PropTypes.object.isRequired
+  },
+  render: function() {
+    var run = this.props.run;
+
+    var labelSpecs = [
+      {path: ['tumor_bam' , 'name'], usePathValue: true, title: 'Tumor BAM', cssClass: 'linked-bam'},
```
Comment (ihodes) [inline cycledash/static/js/runs/components/RunsPage.js]: You no longer need usePathValue or cssClass, which simplifies a good amount of the below logic, too.

**Comment by ihodes** [comment]
Looks good to me--a few comments to be addressed, but the approach is good. :+1:

File: cycledash/static/js/runs/components/RunsPage.js (hunk 1 above)
Comment (jaclynperrone) [inline cycledash/static/js/runs/components/RunsPage.js]: done!

File: cycledash/static/js/runs/components/RunsPage.js (hunk 2 above)
Comment (jaclynperrone) [inline cycledash/static/js/runs/components/RunsPage.js]: done!

File: cycledash/static/js/runs/components/RunsPage.js (hunk 3 above)
Comment (jaclynperrone) [inline cycledash/static/js/runs/components/RunsPage.js]: done!

File: cycledash/static/js/runs/components/RunsPage.js (hunk 4 above)
Comment (jaclynperrone) [inline cycledash/static/js/runs/components/RunsPage.js]: Good call. I removed `usePathValue` from both the status and linked bam components.

File: cycledash/static/js/runs/components/RunsPage.js [hunk 5]
```diff
@@ -386,6 +386,36 @@ var RunLabels = React.createClass({
...
+    run: React.PropTypes.object.isRequired
+  },
+  render: function() {
+    var run = this.props.run;
+
+    var labelSpecs = [
+      {path: ['tumor_bam' , 'name'], title: 'Tumor BAM' },
+      {path: ['normal_bam', 'name'], title: 'Normal BAM' },
+    ];
+    var run_bams = labelSpecs.map(
```
Comment (ihodes) [inline cycledash/static/js/runs/components/RunsPage.js]: `run_bams` => `runBams`

**Comment by ihodes** [comment]
Looks good--one minor comment & you're good to merge.

**Pull request merged**